lib/rack/cache/meta_store.rb

```diff
@@ -81,7 +81,11 @@ def store(request, response, entity_store)
       headers.delete 'Age'
 
       entries.unshift [stored_env, headers]
-      write key, entries
+      if request.env['rack-cache.use_native_ttl'] && response.fresh?
```

Comment:
ah, did not know that, sounds good then!

On Tue, Feb 2, 2016 at 12:38 AM, Julien Letessier <notifications@github.com>
wrote:

> In lib/rack/cache/meta_store.rb
> <https://github.com/rtomayko/rack-cache/pull/117#discussion_r51538218>:
>
> > @@ -81,7 +81,11 @@ def store(request, response, entity_store)
> >        headers.delete 'Age'
> >
> >        entries.unshift [stored_env, headers]
> > -      write key, entries
> > +      if request.env['rack-cache.use_native_ttl'] && response.fresh?
>
> would this be better as an option when creating the store ?
>
> It might, but I was trying to keep this PR about the bug fix — the
> rack-cache.use_native_ttl env field was already present in other parts of
> the code (writing to the entity store
> <https://github.com/rtomayko/rack-cache/blob/master/lib/rack/cache/meta_store.rb#L60>,
> just above in this same file).
> It just wasn't used properly with metastore writes, and wasn't documented.
>
> I'm happy to make this a store option, but as that would be a breaking
> change (some users have code that reliad on use_native_ttl), maybe it'd
> be a better fit in a further PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rtomayko/rack-cache/pull/117/files#r51538218>.
>